docs/tempo/website/traceql/query-editor.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TraceQL query editor
+menuTitle: TraceQL query editor
+description: Learn how to use the TraceQL query editor
+aliases:
+  - /docs/tempo/latest/traceql/construct-query
+weight: 100
+keywords:
+  - Tempo query language
+  - query language
+  - TraceQL
+---
+
+# TraceQL query editor
+
+With Tempo 2.0, you can use the TraceQL query editor in the Tempo data source to build queries and drill-down into result sets. The editor is available in Grafana’s Explore interface.
+
+This screenshot shows the query and results from the search above, `{ .http.method = "POST" } | by(.http.target) | count() > 2`.
```

Comment:
by() doesn't work. remove from here and any screenshots

Reply:
I removed the sentence from the doc and edited the screenshot so the query now reads { .http.method = "POST" } | count() > 2.